ui/src/logs/containers/LogsPage.tsx

```diff
@@ -497,14 +491,14 @@ class LogsPage extends Component<Props, State> {
     return _.get(this.props.logConfig, 'severityLevelColors', [])
   }
 
-  private handleChangeLiveUpdatingStatus = (): void => {
+  private handleChangeLiveUpdatingStatus = async (): Promise<void> => {
     const {liveUpdating} = this.state
 
     if (liveUpdating === LiveUpdating.Play) {
       this.setState({liveUpdating: LiveUpdating.Pause})
       clearInterval(this.interval)
     } else {
-      this.startUpdating()
+      this.handleChooseRelativeTime(0)
```

Comment:
could this `0` be a constant to give context as to why we're passing in `0` here?